Week3/script.js

```diff
@@ -0,0 +1,278 @@
+// JS1 Homework Week 3
+
+// 1. Strings!
+// 1.1 Add the string to your file and log it.
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+console.log(myString);
+
+// 1.2 Log the length of myString.
+console.log(`The length of myString is: ` + myString.length);
```

Comment:
If you are using backticks ` `` `, you can use them like this ` The length of myString is: ${myString.length} `. You don't have to worry about long strings with a lot of quotes. Read this page again: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals